unity-client/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/AssetBundles/AB/AssetPromise_AB.cs

```diff
@@ -42,7 +42,14 @@ protected override bool AddToLibrary()
                 return false;
             }
 
+            if (asset == null)
+            {
+                Debug.LogWarning($"Asset is null when trying to add it to the library: hash == {this.GetId()}");
+                return false;
+            }
+
             asset = library.Get(asset.id);
+            //yep
```

Comment:
```suggestion
            
```

Reply:
Dont mind this PR is just a testing one to pinpoint the source of our crashes. It will be closed